src/platform/packages/shared/kbn-esql-utils/src/utils/query_parsing_helpers.ts

```diff
@@ -39,6 +39,44 @@ export function getIndexPatternFromESQLQuery(esql?: string) {
   return indices?.map((index) => index.name).join(',');
 }
 
+export function getRemoteClustersFromESQLQuery(esql?: string): string[] | undefined {
+  if (!esql) return undefined;
+  const { root } = Parser.parse(esql);
```

Comment:
```suggestion
  const { ast } = EsqlQuery.fromSrc(esql);
```

Reply:
Hmm I thought Vadim had marked Parser.parse as deprecated but I see that it isn't... weird. Disregard.